definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -1 +1,48 @@
-[]
+[
+  {
+    "LiveFrom": "05/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "reissueFromOverdue",
```

Comment:
I think "reissueFromOverdue" should be left on its own as it's the only "reissue" event I think that has a page and fields inside. The others do not need one.

Reply:
@Dizrupto Seeing the `reissueFromOverdue` would not be available anymore when this is release as it's all been merged with the new `reissue` event how do i play this then. As it would mean that this event would not even exist

Reply:
@Dizrupto  Please have a look at the `CaseEventsToFields-xxx-nonprod` , is this not the best path?

Reply:
Yep I think that one shouldn't be merged with the "reissue" event because it has fields that would not be relevant from other states.
<img width="718" alt="Screenshot 2020-11-03 at 15 42 03" src="https://user-images.githubusercontent.com/16866068/98007330-2af2b000-1deb-11eb-82ce-2d1645e8d3bd.png">